.github/actions/deploy-lifecycle-manager-e2e/action.yml

```diff
@@ -151,13 +151,6 @@ runs:
         klm_image_repo: ${{ inputs.klm_image_repo }}
     - name: Expose Metrics Endpoint
       working-directory: lifecycle-manager
-      if: ${{ matrix.e2e-test == 'kyma-metrics' ||
```

Comment:
To Reviewer:

I expose metrics endpoint for all e2e tests, it's not bring too much value to restrict it to certain tests.